atc/creds/vault/manager.go

```diff
@@ -95,6 +96,8 @@ func (manager *VaultManager) Config(config map[string]interface{}) error {
 	manager.PathPrefix = "/concourse"
 	manager.Auth.RetryMax = 5 * time.Minute
 	manager.Auth.RetryInitial = time.Second
+	manager.LoginTimeout = 60 * time.Second
+	manager.QueryTimeout = 60 * time.Second
```

Comment:
I was originally thinking that 60 seconds was a bit conservative, but looks like that's Vault's default timeout, so 👍 :

https://github.com/hashicorp/vault/blob/acecc949078f907e1daabe44712e37964b397687/api/client.go#L151